src/main/java/com/google/edith/DealItem.java

```diff
@@ -121,4 +163,8 @@ public Double getWeight() {
   public String getComment() {
     return comment;
   }
+
+  public String getExpiration() {
+    return expiration;
+  }
```

Comment:
Since expiration is a string could we add a javadoc/example for what the caller should expect here? Is it a string expiration date or time until expiration?